fe/.eslintrc.yml

```diff
@@ -58,5 +68,3 @@ rules:
   react-hooks/rules-of-hooks: off
   react/require-default-props: off
   react/jsx-props-no-spreading: off
-
-  
```

Comment:
These lines didn't deserve to be deleted ):

Reply:
Yep, already added them back. :)